sway/commands/workspace.c

```diff
@@ -209,16 +209,17 @@ struct cmd_results *cmd_workspace(int argc, char **argv) {
 				ws = workspace_create(NULL, name);
 				free(name);
 			}
+			if(ws && auto_back_and_forth){
```

Comment:
Style: missing space between `if` and `(`, missing space betwen `)` and `{`

Reply:
Seems that my editor didn't automatically pick up the `.clang-format`. I should fix that. (I was going to suggest adding one, only to realize it's already there :+1: )

Reply:
Ah, right, I even forgot we had one. It's probably outdated and not complete, PRs welcome to improve it.